variants/coldwar/games_test.go

```diff
@@ -0,0 +1,285 @@
+package coldwar
```

Comment:
This file is in a lot of variants now. I didn't really notice before, but it's pretty much exactly the same in each variant right?

Could you extract it to a variants/testing package or something, so it could be reused instead of copied? :)